clients/hadoopfs/README.md

```diff
@@ -6,3 +6,11 @@ It uses the lakeFS server for metadata operations only.
 ## Publishing
 
 Follow the [HadoopFS release checklist](https://github.com/treeverse/dev/blob/main/pages/lakefs-clients-release.md#lakefs-hadoop-filesystem)
+
+## Building
+
+Maven must be run with Java 8.
+
+## Testing
+
+Maven tests must be run with Java 8.
```

Comment:
Suggest to merge the two and write that the project using Maven, with a prerequisite of having Java 8 installed.